roles/download/defaults/main.yml

```diff
@@ -253,10 +253,10 @@ cilium_init_image_repo: "docker.io/cilium/cilium-init"
 cilium_init_image_tag: "2019-04-05"
 cilium_operator_image_repo: "docker.io/cilium/operator"
 cilium_operator_image_tag: "{{ cilium_version }}"
-kube_ovn_db_image_repo: "index.alauda.cn/alaudak8s/kube-ovn-db"
-kube_ovn_node_image_repo: "index.alauda.cn/alaudak8s/kube-ovn-node"
-kube_ovn_cni_image_repo: "index.alauda.cn/alaudak8s/kube-ovn-cni"
-kube_ovn_controller_image_repo: "index.alauda.cn/alaudak8s/kube-ovn-controller"
+kube_ovn_db_image_repo: "kubeovn/kube-ovn-db"
```

Comment:
this needs to be prepended with `docker.io/kubeovn/kube-ovn-db`

Reply:
done